targets/TARGET_STM/TARGET_STM32F3/device/stm32f3xx_hal_adc_ex.c

```diff
@@ -5660,12 +5660,6 @@ HAL_StatusTypeDef HAL_ADC_ConfigChannel(ADC_HandleTypeDef* hadc, ADC_ChannelConf
       /* Set handle of the other ADC sharing the same common register         */
       ADC_COMMON_ADC_OTHER(hadc, &tmphadcSharingSameCommonRegister);
       
```

Comment:
why is it removed ? Is it a patch that we have added previously ?

Reply:
yes, same kind of patch as #6412